.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
To be removed. They wont be submitting templates to us.

Reply:
removed it.